pages/windows/bcdboot.md

```diff
@@ -0,0 +1,24 @@
+# bcdboot
+
+> A system utility to configure boot files or repair them in Windows.
```

Comment:
The page is in the `windows` directory, no need to explicitly state that in the description.
```suggestion
> A system utility to configure or repair boot files.
```

Reply:
yep, LGTM